### DISCUSSION TITLE
how create custom message in Validator?

### DISCUSSION THREAD

**Discussion by sasha172017** [adonisjs/core#1843] [Help]
how create custom message in Validator and how make translation?

**Comment by hariprasath-yadav** [comment]
Refer to this page if you use V5
https://preview.adonisjs.com/guides/http/exception-handling

**Reply by DraftProducts** [reply]
I'm not sure about it @hariprasath-yadav.
Here the right page https://preview.adonisjs.com/guides/validator/custom-messages

**Reply by hariprasath-yadav** [reply]
Your right @DraftProducts but I used
http exception-handling with
https://preview.adonisjs.com/guides/auth/handling-exceptions
on my project to for authentication custom messages, so i suggest it
